### DISCUSSION TITLE
Prevent page scroll on middle mouse click drag

### DISCUSSION THREAD

**Discussion by hanhanxue** [jeffreylanters/react-unity-webgl#562] [Question and Answer]
Hello! I have an interaction bound to middle mouse click and drag in my Unity instance. However on the webpage that also scrolls the page. I do I prevent middle mouse click and drag on Unity canvas from scrolling the webpage while still keeping the interaction inside the Unity instance?

Thanks!

**Comment by jeffreylanters** [comment]
Hi! When building applications for the web, you're basically running your application inside of another application. Your browser has the most power here, so its keybindings have right of way. As far as I know there is no way to overwrite such bindings.

**Reply by hanhanxue** [reply]
Thanks for the reply @jeffreylanters! I was able to solve this by storing parent div to <Unity /> component in a ref and preventing default behavior for middle mouse move events on that ref

**Reply by jeffreylanters** [reply]
Oh that's even better, I really thought it wasn't possible. I've learned something new! Thanks for sharing.
